src/demos/multi_file_viewer_demo.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div class="multi-file-viewer-demo">
+
+    <div>
+      <p @click="open_file(1)"> Add A </p>
+      <p @click="open_file(2)"> Add B </p>
+      <p @click="open_file(3)"> Add C </p>
+      <p @click="open_file(4)"> Add D </p>
+      <p @click="open_file(5)"> Add E </p>
+      <div class="mvf-container">
+        <multi-file-viewer
+          ref="multi_file"
+          :height_of_view_file="height_of_view_file">
+        </multi-file-viewer>
+      </div>
+    </div>
+
+    <div class="faux-incorrect-row"> </div>
+    <div class="faux-correct-row"> </div>
+    <div class="faux-unavailable-row"> </div>
+
+
+    <div>
+      <p class="letters" @click="open('A')"> Add A </p>
+      <p class="letters" @click="open('B')"> Add B </p>
+      <p class="letters" @click="open('C')"> Add C </p>
+      <div class="mvf-container">
+        <multi-file-viewer
+          ref="multi_file2">
+        </multi-file-viewer>
+      </div>
+    </div>
+
+  </div>
+</template>
+
+<script lang="ts">
+
+  import MultiFileViewer from '@/components/multi_file_viewer.vue';
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+
+  @Component({
+    components: { MultiFileViewer }
+  })
+  export default class MultiFileViewerDemo extends Vue {
+
+    open_file(file_number: number) {
```

Comment:
This seems like a swiss-army-knife function. I know it's just a demo, but you could do something with arrays or dictionaries to avoid all that iffing.